src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -42,6 +42,11 @@ public class MasterTestRunner : ITestRunner
         private ITestRunnerFactory _testRunnerFactory;
         private bool _disposed;
 
+        private TestEventDispatcher _eventDispatcher = new TestEventDispatcher();
+        private WorkItemTracker _workItemTracker = new WorkItemTracker();
```

Comment:
I don't think this is plugged in, is it? As in, it's not been added been added as an event listener. They're not detected automatically or anything, are they?

Reply:
I remember adding it but I don't see it... could have removed it "temporarily" in order to test something. I'll check.

Reply:
Good catch! It was in but I must have removed it for a test and forgot to put it back... see start of `MasterTestRunner.RunTests'